doc/man1/ts.pod

```diff
@@ -557,10 +557,10 @@ OID section of the config file):
 
 Before generating a response a signing certificate must be created for
 the TSA that contains the B<timeStamping> critical extended key usage extension
-without any other key usage extensions. You can add the
-'extendedKeyUsage = critical,timeStamping' line to the user certificate section
-of the config file to generate a proper certificate. See L<req(1)>,
-L<ca(1)>, L<x509(1)> for instructions. The examples
+without any other key usage extensions. You can add this line to the
+user certificate section of the config file to generate a proper certificate;
+   extendedKeyUsage = critical,timeStamping
```

Comment:
You must have a blank line before and after this one. Remember, blank lines are block separators in the pod format.